src/apidocs/ApiDocsArea.tsx

```diff
@@ -224,4 +221,18 @@ const BaseClasses = ({ value }: { value: ApiDocsBaseClass[] }) => {
   );
 };
 
+const DocString = ({ value }: { value: string }) => {
+  const html = renderMarkdown(value);
+  console.log(value);
+  return (
+    <Text
```

Comment:
There's an issue with newlines that are \n in the reST. See the Image docs.

Reply:
I'll track this separately too, it's already broken when it comes back from Pyright.